### PR TITLE
Add unit tests on CommandLockHandler

### PR DESCRIPTION
**Pull request by clement-michelet**

### CODE REVIEW

**Comment by juchi** [comment]
:+1:

File: Tests/Handler/CommandLockHandlerTest.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<?php
+
+
```
Comment (glenouvel) [inline Tests/Handler/CommandLockHandlerTest.php]: empty line to suppress

Comment (clement-michelet) [inline Tests/Handler/CommandLockHandlerTest.php]: fixed

**Comment by clement-michelet** [comment]
ping @glenouvel

**Comment by glenouvel** [comment]
:+1:

**Pull request merged**